config/m4/gdrcopy.m4

```diff
@@ -48,7 +48,11 @@ AS_IF([test "x$with_gdrcopy" != "xno"],
         AS_IF([test "x$gdrcopy_happy" = "xyes"],
             [
                 AC_SUBST(GDR_COPY_CPPFLAGS, "-I$ucx_check_gdrcopy_dir/include/ ")
-                AC_SUBST(GDR_COPY_LDFLAGS, "-lgdrapi -L$ucx_check_gdrcopy_dir/lib64")
+
+                gdr_copy_ldflags="-lgdrapi "
+                AS_IF([test ! -z "$ucx_check_gdrcopy_libdir"],
+                    gdr_copy_ldflags += "-L$ucx_check_gdrcopy_libdir ")
```

Comment:
not sure += is portable
better do `gdr_copy_ldflags="$gdr_copy_ldflags -L$ucx_check_gdrcopy_libdir"`

Reply:
Fixed